docs/2-cloud-storage-providers.md

```diff
@@ -4,37 +4,10 @@ One of the core feature of Flysystem is its ability to interact easily with remo
 including many cloud storage providers. This bundle provides the same level of support for these
 cloud providers by providing corresponding adapters in the configuration.
 
-* [Azure](#azure)
 * [AsyncAws S3](#asyncaws-s3)
 * [AWS S3](#aws-s3)
 * [DigitalOcean Spaces](#digitalocean-spaces)
 * [Scaleway Object Storage](#scaleway-object-storage)
-* [Google Cloud Storage](#google-cloud-storage)
```

Comment:
We are removing quite useful adapters here (Azure and Google Cloud Storage), are there community alternatives?

Reply:
All Azure adapters doesn't even have issues for flysystem v2.

I just found that league published google cloud adapter recently (https://github.com/thephpleague/flysystem-google-cloud-storage) and I will work towards this adapter soon and will update PR.

Reply:
Let's implement the official Google Cloud Storage package yes, and let's ignore Azure for now: if people need it, it'll be added back.

Reply:
OK, I have already done that :)